### ISSUE TITLE
JAWS is not announcing the role for menu items present on the page.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1029]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1217576

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" Menu item and select it.
Navigate through all the controls under "Data Explorer" blade.
Turn on JAWS and navigate to menu items like 'New collection', 'Enable azure synapse link', 'Enable notebooks', 'settings', 'Feedback' and 'Open full screen'.
Observe the screen reader behavior.
Verify the issue.

Actual Result:
JAWS is not announcing the role for menu items(New collection, Enable azure synapse link, Enable notebooks, settings, Feedback, and Open full screen) present on the page. It reads only the name as well as count and position information.

Expected Result:
JAWS should announce the role for menu items(New collection, Enable azure synapse link, Enable notebooks, settings, Feedback, and Open full screen) present on the page. It should read the name role as menu item as well as count and position information for each of them.


Note:
Similar issue exits with NVDA but not narrator.

https://user-images.githubusercontent.com/8519407/132720602-2ab7b616-c85a-47f0-8fa1-c7e32defadd5.mp4
